data/core/src/commonMain/kotlin/com/tunjid/heron/data/repository/TimelineRepository.kt

```diff
@@ -239,9 +243,43 @@ internal class OfflineTimelineRepository(
         savedStateDataSource.savedState
             .map(SavedState::signedProfilePreferencesOrDefault)
 
-    override fun labelers(): Flow<List<Labeler>> =
-        // TODO: Get labeler from bluesky moderation service
-        flowOf(Collections.DefaultLabelers)
+    override fun labelers(): Flow<CursorList<Labeler>> =
```

Comment:
This has been implemented in this PR: https://github.com/tunjid/heron/pull/613/files

The labelers fetched are those from the user's preferences.

Reply:
Yeah, close it.